src/Table/elements.js

```diff
@@ -0,0 +1,171 @@
+import styled, { css } from 'styled-components';
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: ${({ width }) => width};
+  ${({ height }) => height && css` height ${height}`};
+
+  box-shadow: 0 1px 0 0 hsla(0, 0%, 0%, 0.06);
+`;
+
+export const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+`;
+
+export const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+`;
+
+const getCellFlex = width => (width === parseInt(width).toString() ? width : 'initial');
```

Comment:
Please add comments for functions

Reply:
@leopoldhoudin can you help me for this ?

Reply:
```javascript
/**
 * Gets the value of flex attribute for a column.
 * 
 * If the given width is an integer, this means that cells width are given in weights relative to each other, hence the 
 * resulting flex value (in a container with flex direction set to row), should be the weight (the given integer).
 * If the given value is a width as a string (for instance with px or rem), the flex attribute shouldn't be used (because
 * width attribute is used in place), and thus the flex value is set to initial.
 *
 * @param {string} width - The width of the column.
 *
 * @returns {string} - The value of the CSS flex attribute.
 */
```